babel.config.js

```diff
@@ -123,18 +19,4 @@ module.exports = {
     'lodash',
     '@babel/plugin-proposal-class-properties',
   ],
-  env: {
-    esm: {
-      plugins: [patternflyTransformImports('esm'), fecTransformImports('esm')],
-    },
-    production: {
-      plugins: [patternflyTransformImports('esm'), fecTransformImports('esm')],
-    },
-    development: {
-      plugins: [patternflyTransformImports('esm'), fecTransformImports('esm')],
-    },
-    componentTest: {
-      plugins: ['istanbul'],
-    },
```

Comment:
I think this may still be required for cypress to collect coverage, or?

Reply:
Added it back